packages/actor-optimize-query-operation-filter-pushdown/lib/ActorOptimizeQueryOperationFilterPushdown.ts

```diff
@@ -75,8 +84,9 @@ export class ActorOptimizeQueryOperationFilterPushdown extends ActorOptimizeQuer
       repeat = false;
       operation = Util.mapOperation(operation, {
         filter(op: Algebra.Filter, factory: Factory) {
+          const comunicaFunctions = action.context.get(KeysInitQuery.extensionFunctions);
```

Comment:
Instead of `comunicaFunctions`, I'd just use something like `extensionFunctions` everywhere.